src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -4,6 +4,13 @@
 public class Task10 {
 
     public static boolean containDigitTwo(int n) {
+
+        while (n > 0) {
+            if (n % 10 == 2)
+            if (n / 10 == 10) {
+                return true;
+            }
+        }
```

Comment:
It's not correct solution.
It'll loop forever for some numbers, for example, when `n=27`:
1) `n>0` -> `true` -> go into while loop
2) `n%10 == 2` -> `false` -> do not divide `n` by `10` -> go to step 1.